README.md

```diff
@@ -122,28 +129,32 @@ The interaction with OpenShift is via the command line tool _oc_ which is copied
            To login as administrator:
                oc login -u system:admin
 
-    Note that, the IP seen above is dynamic and can change. It can be retrieved with `minishift ip`. Also,
-    instead of 'kvm', you will see 'xhyve' on Mac OS and 'hyperv' on Windows.
-
-    **Note:** By default Minishift uses the driver most relevant to the host OS.
-To use a driver of choice for Minishift, use the `--vm-driver=xxx` flag with `minishift start`. For example, to use VirtualBox instead of KVM for Fedora, use `minishift start --vm-driver=virtualbox`.
+  **Note**:
 
-1. Add `oc` binary to the _PATH_:
+  - The IP is dynamically generated for each Minishift instance. To check the IP, run the `minishift ip` command.
+  - By default Minishift uses the driver most relevant to the host OS. To use a different driver, set the `--vm-driver` flag in `minishift start`. For example, to use VirtualBox instead of KVM on Linux operating systems, run `minishift start --vm-driver=virtualbox`. For more information about `minishift start` options,
+  see the [minishift start command reference](/docs/minishift_start.md).
 
-     **Note:** How to modify the _PATH_ varies depending on host OS, version of the OC binary and
-     other variables. In case of doubt, you can check the content of the
-     `~/.minishift/cache/oc` directory.
+2. Add the `oc` binary to the _PATH_ environment variable. For example, on Linux
+operating systems, run the following command:
 
         $ export PATH=$PATH:~/.minishift/cache/oc/v1.3.1
 
-1. Login to your OpenShift account and authenticate yourself:
+  **Note:** Depending on the operating system and the `oc` version, you might need
+  to use a different command to add the path to the _PATH_ environment variable.
+  To verify the `oc` version, check the contents of the `~/.minishift/cache/oc` directory.
 
-        $ oc login https://192.168.99.128:8443 -u developer -p developer
+3. You can access the OpenShift Web console from your Web browser by navigating to the IP address listed in the CLI with the same credentials that you use to log
```

Comment:
The original step to log into the console was deemed redundant yesterday when @Preeticp was testing things, so we said we will delete it. Since the output of minishift start specifically states that you can log in to the console by navigating to the URL, I thought I would write this here. If it's unrelated to the quickstart I can remove it.